index/constraints.txt

```diff
@@ -1,41 +1,40 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
 #
-#    pip-compile --extra-index-url=https://packages.dea.ga.gov.au/ --output-file=constraints.txt requirements.txt
+#    pip-compile --extra-index-url=https://packages.dea.ga.gov.au/ --output-file=constraints.txt --pre requirements.txt
```

Comment:
I don't think you want the `--pre` here, it looks like it accepts pre-release versions of packages.

Reply:
I was following instructions in the README. I update it to not use `--pre` for releases?
https://github.com/opendatacube/datacube-docker/tree/main/index#how-to-create-a-new-image-with-latest-odc-tools